analyticsdataserver/settings/base.py

```diff
@@ -311,6 +311,10 @@
 DEFAULT_PAGE_SIZE = 25
 MAX_PAGE_SIZE = 100
 
+# Only used in a local developer tools script. Override in local.py
+LMS_BASE_URL = 'http://localhost:8000/'
```

Comment:
I'm not sure, but it might make more sense for these to be set to `None` here and then with these default settings in `local.py`.  Just a thought!